worlds/zillion/__init__.py

```diff
@@ -118,16 +118,22 @@ def flush(self) -> None:
     """
     my_locations: List[ZillionLocation] = []
     """ This is kind of a cache to avoid iterating through all the multiworld locations in logic. """
-    slot_data_ready: threading.Event
-    """ This event is set in `generate_output` when the data is ready for `fill_slot_data` """
+    finalized_gen_data: Optional[GenData]
+    """ Finalized generation data needed by `generate_output` directly and by `fill_slot_data` indirectly. """
```

Comment:
```suggestion
    """ Finalized generation data needed by `generate_output` and by `fill_slot_data`. """
```
to go with the other change